### PR TITLE
ROCm3.9 recipe for rocblas, hipblas & rocsolver

### PR DESCRIPTION
**Pull request by gkathirv**

### CODE REVIEW

**Comment by gkathirv** [comment]
@adamjstewart @srekolam @arjun-raj-kuppala

File: var/spack/repos/builtin/packages/rocblas/package.py [hunk 1]
```diff
@@ -59,9 +60,14 @@ class Rocblas(CMakePackage):
...
 
+    resource(name='Tensile',
+             git='https://github.com/ROCmSoftwarePlatform/Tensile.git',
+             commit='b68edc65aaeed08c71b2b8622f69f83498b57d7a',
+             when='@3.9.0')
+
     # Status: https://github.com/ROCmSoftwarePlatform/Tensile/commit/a488f7dadba34f84b9658ba92ce9ec5a0615a087
     # Not yet landed in 3.7.0, nor 3.8.0.
-    patch('0001-Fix-compilation-error-with-StringRef-to-basic-string.patch')
+    #patch('0001-Fix-compilation-error-with-StringRef-to-basic-string.patch')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/rocblas/package.py]:
```suggestion
    patch('0001-Fix-compilation-error-with-StringRef-to-basic-string.patch', when='@:3.8')
```

**Review by srekolam — APPROVED** [review]

File: var/spack/repos/builtin/packages/rocblas/package.py [hunk 2]
```diff
@@ -13,8 +13,9 @@ class Rocblas(CMakePackage):
     homepage = "https://github.com/ROCmSoftwarePlatform/rocBLAS/"
     url      = "https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-3.5.0.tar.gz"
 
-    maintainers = ['haampie']
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/rocblas/package.py:16]: @haampie are you okay with this? We could also have all 3 of you as maintainers on all rocm packages.

Comment (haampie) [reply var/spack/repos/builtin/packages/rocblas/package.py:16]: I'm fine with this

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**